calico_containers/pycalico/block.py

```diff
@@ -516,6 +556,16 @@ def get_block_cidr_for_address(address):
     return IPNetwork(block_id)
 
 
+def validate_block_size(cidr):
+    """
+    Check that the CIDR block size is valid.  This checks that it is at least
+    as large as the minimum block size.
+    """
+    assert isinstance(cidr, IPNetwork)
+    #TODO Should probably limit the maximum block size too.
```

Comment:
Are we planning to do this TODO in this PR?

If not we should probably remove the TODO and just raise an issue.